src/eslint-rules/enforce-result-pattern.js

```
@@ -0,0 +1,121 @@
+          });
+        }
+      },
+
+      // :A: api check for try-catch in async functions
+      TryStatement(node) {
+        const func = context.getAncestors().reverse().find(
+          ancestor => ancestor.type === 'FunctionDeclaration' || 
+                     ancestor.type === 'ArrowFunctionExpression'
+        );
+        
+        if (func && func.async) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)